### PR TITLE
fix missing attributes when register plugin

### PR DESCRIPTION
**Pull request by jasonyihk**
fix missing attributes error when register plugin with HapiJS: ` "[1] "attributes" is required"` BY:
- add/export instrument for plugin registration

e.g.
const server = new hapi.Server();
const promster = require('@promster/hapi');
promster.instrument(server, options);

related to issue: https://github.com/tdeekens/promster/issues/131

### CODE REVIEW

File: packages/hapi/modules/index.js [hunk 1]
```diff
@@ -23,3 +23,6 @@ exports.getContentType = getContentType;
 exports.Prometheus = Prometheus;
 exports.defaultRegister = defaultRegister;
 exports.defaultNormalizers = defaultNormalizers;
+exports.instrument = (server, options) => {
```
[the remark this answers is not in the document]
Comment (jasonyihk) [reply packages/hapi/modules/index.js]: won't change the existing register method, can still use
`app.register(createPlugin({ options }));`
. exported instrument method provides another option to register the plugin with hapijs.

[the remark this answers is not in the document]
Comment (jasonyihk) [reply packages/hapi/modules/index.js]: will add unit test later. stay tuned.

**Comment by tdeekens** [comment]
Changes:
 - @promster/express: 2.2.1 => 2.2.2
 - *@promster/hapi: 2.2.0 => 2.2.1*
 - @promster/marblejs: 1.0.0 => 1.0.1
 - @promster/metrics: 2.2.0 => 2.2.1
 - @promster/server: 2.1.0 => 2.1.1

Thanks. Enjoy :)

File: packages/hapi/modules/index.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (tdeekens) [reply packages/hapi/modules/index.js]: I guess I read that too late. I assume you wanted a release already. Sorry bout that. Feel free to open another PR :)

**Comment by tdeekens** [comment]
Alright `@promster/hapi@2.2.3` should actually work as expected.

**Pull request merged**